lib/jsonapi_compliable/adapters/abstract.rb

```diff
@@ -488,6 +497,19 @@ def update(model_class, update_params)
       def destroy(model_class, id)
         raise 'you must override #destroy in an adapter subclass'
       end
+
+      private
+
+      def activerecord_adapter
+        ::JsonapiCompliable::Adapters::ActiveRecord::Base.new
+      end
+
+      def activerecord_associate?(parent, child, association_name)
+        defined?(::ActiveRecord) &&
```

Comment:
Might make sense to memoize based on input. Can see potential perf issues due to reflections use.